core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -125,6 +125,20 @@ public ServerMain(String[] args) throws Exception {
                 break;
         }
         log.advisory().$("available CPUs: ").$(Runtime.getRuntime().availableProcessors()).$();
+        log.advisory().$("db root: ").$(configuration.getCairoConfiguration().getRoot()).$();
+        try (Path path = new Path()) {
+            path.of(configuration.getCairoConfiguration().getRoot()).$();
```

Comment:
There is dependency here that path will be 255 chars and implementation of that can change in the future.

Reply:
Path is currently allocated 256 bytes unless capacity is specified lower:

![image](https://user-images.githubusercontent.com/7276403/147139884-f149ac27-8101-407c-adee-1bce655c8858.png)

There is no dependency between path to the database and returned file system name